test/new_tests/test_is_connected.py

```diff
@@ -60,10 +61,10 @@ def test_pos_is_connected_after_multiple_connects(self):
 
     def test_is_connected_after_close(self):
         """
-        Test that is_connected returns False after a successful calls to
-        connect() and close()
+         Client call itself establishes connection.
+         Connect/Close are deprecated and it is no-op to client
         """
         self._connect()
         assert self.client.is_connected() is True
         self.client.close()
-        assert self.client.is_connected() is False
+        assert self.client.is_connected() is True
```

Comment:
No-oping `close()` might be a dangerous change because the user assumes the `close()` is successful even when the connection wasn't closed.